### DISCUSSION TITLE
TODO and XXX highlighted but not FIXME

### DISCUSSION THREAD

**Discussion by mooreye** [nvim-treesitter/nvim-treesitter#6250] [Q&A] [answered]
I have the following in `init.vim`:
```
lua << EOF
require('nvim-treesitter.configs').setup {
  highlight = {
    enable = true,
    additional_vim_regex_highlighting = false
  }
}
vim.api.nvim_set_hl(0, '@text.note', { link = 'Todo' })
EOF
```

Due to this, it highlights `XXX` and `TODO` keywords in comments, but not `FIXME`. Once I remove this part from `init.vim`, it highlights all 3 of them. How do I get `FIXME` to be highlighted with treesitter?

**Comment by clason** [comment, marked as answer]
`FIXME` is highlighted as `@comment.error`; see `queries/comment/highlight.scm`.
